src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -207,7 +223,8 @@ private static void validateTableAccess(Client engineClient, JSONObject mapping,
             }
 
             if (hasAccess) {
-                String authenticateWith = String.format("aadapp=%s;%s", config.getAuthAppid(), config.getAuthAuthority());
+                String authenticateWith = String.format("aadapp=%s;%s", config.getAuthAppId(),

```

Comment:
I double checked, this works fine with both Azure AD Application and Managed Identities